resources/dicts/events/misc_events/beach.json

```diff
@@ -1,118 +1,53 @@
-                "apprentice"
-            ]
-        }
-    },
-    {
-        "event_id": "bch_misc",
-        "biome": [
-            "beach"
-        ],
-        "camp": [
-            "any"
-        ],
-        "season": [
-            "any"
-        ],
-        "tags": [],
-        "weight": 20,
-        "event_text": "As a storm rages across the beach and waves crash upon the shore, m_c is tucked up in the medicine cat den, {PRONOUN/m_c/poss} sleep anything but restful. {PRONOUN/m_c/subject/CAP} {VERB/m_c/keep/keeps} seeing flashes of prophecy_list_sight. {PRONOUN/m_c/subject/CAP} {VERB/m_c/wake/wakes} with a shiver, wondering what {PRONOUN/m_c/poss} dreams mean for the Clan.",
-        "m_c": {
-            "age": [
-                "young adult",
-                "adult",
-                "senior adult",
-                "senior"
-            ],
-            "status": [
-                "medicine cat"
-            ],
-            "skill": [
-                "PROPHET,1"
-            ]
-        }
-    },
-    {
-        "event_id": "bch_misc",
-        "biome": [
-            "beach"
-        ],
-        "camp": [
-            "any"
-        ],
-        "season": [
-            "any"
-        ],
-        "tags": [],
-        "weight": 20,
-        "event_text": "The things {PRONOUN/m_c/subject} saw today: omen_list_sight... m_c dips a paw in a rockpool, watching the tiny, crawling life inside, and considering what the omens foretell.",
-        "m_c": {
-            "age": [
-                "young adult",
-                "adult",
-                "senior adult",
-                "senior"
-            ],
-            "status": [
-                "medicine cat"
-            ],
-            "skill": [
-                "OMEN,1"
-            ]
-        }
-    },
-    {
-        "event_id": "bch_misc",
-        "biome": [
-            "beach"
-        ],
-        "camp": [
-            "any"
-        ],
-        "season": [
-            "any"
-        ],
-        "tags": [
-            "war"
-        ],
-        "weight": 20,
-        "event_text": "m_c sits by the water, contemplating the calm waves that brush {PRONOUN/m_c/poss} paws and wondering why those times are not as peaceful as the tides.",
-        "m_c": {
-            "age": [
-                "young adult",
-                "adult",
-                "senior adult",
-                "senior"
-            ],
-            "status": [
-                "leader"
-            ],
-            "trait": [
-                "thoughtful",
-                "wise",
-                "righteous"
-            ]
-        }
+  {
+    "event_id": "bch_misc_war_any_apps1",
+    "biome": ["beach"],
+    "camp": ["any"],
+    "season": ["any"],
+    "tags": ["war", "clan_apps"],
+    "weight": 15,
+    "event_text": "The apprentices of the Clan are taking special training to learn how to disguise their scents from enemies with kelp and seawater.",
+    "m_c": {
+      "status": ["apprentice"]
```

Comment:
please also include `age` with "any".  both `age` and `status` params should be included with "any" if they would otherwise be empty

Reply:
Should also hopefully be fixed, updated all misc events across the board with the parameters. Ill update the new_cat events shortly (not marking as resolved to remind myself)